client/src/Components/EmployeeProfile/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import './EmployeeProfile.scss';
+import Header from '../Header/Header'
+import Button from '../Button/index'
+ 
+ 
+        const EmployeeProfile = () => {
+            return (
+                <div className="employee-profile">
+                    <header>
+                        <Header />
+                        <img className="employee-profile__image__small" src="/image/user.png" />
+
+                    </header>
+                    
+                    <div className="employee-profile__main">
+                        <img className="employee-profile__image__big" src="/image/user.png" />
```

Comment:
the className convention when you use modifier is : "employee-profile__image employee-profile__image--big"  and  "employee-profile__image employee-profile__image--small"

Reply:
In fact, you don't need a modifier at all for this component. as navigation component will take care of the small profile image. you just need to create the image for the talent profile. And try with the real image not SVG.